### DISCUSSION TITLE
I donot know this is quill-mention's bug or quill's bug?

### DISCUSSION THREAD

**Discussion by CHWYH** [afconsult/quill-mention#284] [Q&A]
![动画](https://user-images.githubusercontent.com/13195923/178925439-f361ba7f-1ebd-4741-b0f8-f09f8fa3f733.gif)

### set spaceAfterInsert=false, I insert Chinese characters after mention, the first character will appear an error ，As the gif shows

**Comment by minzhang2** [comment]
有解了嘛？

**Comment by CHWYH** [comment]
@minzhang2  没有

**Reply by Beamon-project** [reply]
![image](https://github.com/user-attachments/assets/d1ad3ef9-857f-413e-824c-c3618d133d61)


设置allowedChars: /./,
